runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/NpcClueScroll.java

```diff
@@ -24,7 +24,11 @@
  */
 package net.runelite.client.plugins.cluescrolls.clues;
 
+import java.util.List;
+
 public interface NpcClueScroll
 {
 	String getNpc();
+
+	List<String> getNpcs();
```

Comment:
I would probably remove the getNpc() and just use getNpcs everywhere and return 1 item litsts